buildSrc/src/main/groovy/org/elasticsearch/gradle/BuildPlugin.groovy

```diff
@@ -120,6 +120,7 @@ class BuildPlugin implements Plugin<Project> {
                 println "  JDK Version           : ${gradleJavaVersionDetails}"
                 println "  JAVA_HOME             : ${gradleJavaHome}"
             }
+            println " Random Testing Seed    : ${project.testSeed}"
```

Comment:
I think the seed should be generated here. Otherwise eg when a plugin is built using build-tools, but without an ES checkout, this would fail here.